api/app/signals/apps/services/domain/dsl.py

```diff
@@ -108,7 +108,8 @@ def process_routing_rules(self, signal):
                             {
                                 'id': rule._department.id
                             }
-                        ]
+                        ],
+                        'employee': None if not rule.employee else rule.employee
```

Comment:
This should not be part of the `routing_departments` flow. It should be in its own flow.

Reply:
This is added in the process_routing_rules function.
Which then calls the update_routing_departments function in managers.py.

Since we are working with routing rules why wouldn't this be the correct spot to add user assignment?